<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>Параметры</string>
        </property>
        <widget class="QWidget" name="">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>40</y>
           <width>185</width>
           <height>161</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Параметры кривой Безье</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QFormLayout" name="formLayout">
              <item row="0" column="1">
               <widget class="QRadioButton" name="radioButton">
                <property name="text">
                 <string>Составное</string>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">buttonGroup_2</string>
                </attribute>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QRadioButton" name="radioButton_2">
                <property name="text">
                 <string>Простое</string>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">buttonGroup_2</string>
                </attribute>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>Очистить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Замкнуть</string>
            </property>
           </widget>
          </item>
         </layout>
         <zorder>pushButton</zorder>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>767</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="openAction"/>
    <addaction name="saveAction"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Программа</string>
    </property>
    <addaction name="aboutProgramAction"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="openAction">
   <property name="text">
    <string>&amp;Открыть</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="text">
    <string>&amp;Сохранить</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="aboutProgramAction">
   <property name="text">
    <string>&amp;О программе</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
